examples/issue-75/docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
   s3-server:
     image: minio/minio:RELEASE.2019-04-09T01-22-30Z
     volumes:
-      - ./s3-server-cmd:/bin/s3-server-cmd:ro
+      - ./data/s3-server/s3-server-cmd:/bin/s3-server-cmd:ro
```

Comment:
This may be in `./bin/s3-server/` or `/conf/s3-server/` folder.

Reply:
You changed the file mode of `examples/issue-75/data/s3-server/s3-server-cmd` from 755 to 644 (removed execution mode for an executable)

Reply:
Windows issue, thanks! i've searching solution for windows Git mess...